Resources/Prototypes/_Harmony/Traits/speech.yml

```
@@ -247,3 +247,21 @@
         Male: UnisexVox
         Female: UnisexVox
         Unsexed: UnisexVox
+
+- type: trait
+  id: IrishAccent
+  name: trait-irish-accent-name
+  description: trait-irish-accent-desc
+  category: SpeechTraits
+  cost: 1
+  components:
+    - type: IrishAccent
+
+- type: trait
+  id: ScandinavianAccent
+  name: trait-scandinavian-name
+  description: trait-scandinavian-desc
+  category: SpeechTraits
+  cost: 1
+  components:
+    - type: ScandinavianAccent
```

Files should always end with one empty line
```suggestion
    - type: ScandinavianAccent

```